gc/base/GCExtensionsBase.hpp

```diff
@@ -1595,6 +1619,11 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, dnssMinimumContraction(0.0)
 		, enableSplitHeap(false)
 		, aliasInhibitingThresholdPercentage(0.20)
+		, adaptiveGCThreading(false)
+		, adaptiveThreadCount(0)
+		, adaptiveThreadingSensitivityFactor((float)1.0)
+		, adaptiveThreadingWeightActiveThreads((float)0.50)
+		, adaptiveThreadBooster((float)0.85)
```

Comment:
I believe more correct is to do 0.85f

Reply:
changed - I prefer this as well. It's just that I saw the casting style done in the file.